src/common/components/clusterDetail/ProgressBarTexts.tsx

```diff
@@ -1,18 +1,17 @@
-import { Host, HostRole } from '../../api/types';
 import React from 'react';
+import { Host, HostRole } from '../../api/types';
 import { pluralize } from 'humanize-plus';
 import { CheckCircleIcon, ExclamationCircleIcon, InProgressIcon } from '@patternfly/react-icons';
 import { global_danger_color_100 as dangerColor } from '@patternfly/react-tokens/dist/esm/global_danger_color_100';
-import { Stack, StackItem, TextContent } from '@patternfly/react-core';
 import { global_success_color_100 as okColor } from '@patternfly/react-tokens/dist/esm/global_success_color_100';
-import BorderedIcon from '../ui/BorderedIcon/BorderedIcon';
+import ClusterProgressItem from './ClusterProgressItem';
 
 type HostProgressProps = {
   hosts: Host[];
   hostRole: HostRole;
 };
 
-export const ProgressBarTexts: React.FC<HostProgressProps> = ({ hosts, hostRole }) => {
+export const ProgressBarTexts = ({ hosts, hostRole }: HostProgressProps) => {
   const filteredHosts = hosts.filter((host) => host.role && hostRole === host.role);
   const failedHostsCount = filteredHosts.filter((host) => host.status === 'error').length;
   const hostCountText = (hostRole: HostRole) =>
```

Comment:
Cab be declared outside the component, just pass in the `filteredHosts` and `failedHostsCount` as additional arguments.